scripts/qa_crowbarsetup.sh

```diff
@@ -3405,11 +3405,15 @@ function onadmin_proposal
     fi
     local proposal
     for proposal in nfs_client pacemaker database rabbitmq keystone swift \
-        ceph monasca glance cinder neutron nova `horizon_barclamp` \
-        ceilometer heat manila trove barbican magnum sahara murano \
-        aodh tempest; do
+        ceph monasca glance cinder neutron ; do
```

Comment:
why not to split it in two sets: pre-pacemaker+pacemaker, post-pacemaker? In this case we can detect problems in database, rabbitmp, keystone, glance, cinder and neutron, that can be deployed in HA too